nim_status_client.log

```diff
@@ -0,0 +1,2 @@
+INF 2021-11-25 15:22:59.889+02:00 Terminating the app as the second instance topics="main" tid=534335 file=nim_status_client.nim:73
+INF 2021-11-25 21:48:24.385+02:00 Terminating the app as the second instance topics="main" tid=938952 file=nim_status_client.nim:73
```

Comment:
This should probably not be in here

Reply:
Yeah let's add that to the gitignore!

Reply:
removed

> I think more QML can be removed I think the way we display message is different for timeline and as such can be removed too. I believed this is this:
> https://github.com/status-im/status-desktop/blob/master/ui/imports/shared/views/chat/MessageView.qml#L330-L356
> (but please double check ;))

Checking that I see that `chatType: 4` is  `Profile` :
```
type ChatType* {.pure.}= enum
  Unknown = 0,
  OneToOne = 1, 
  Public = 2,
  PrivateGroupChat = 3,
  Profile = 4,
  
```
could somebody confirm that `Profile` corresponds to readonly `property int chatTypeStatusUpdate: 4` from Contants.qml? @PascalPrecht git blames you for the implementation, commit `f512d5063c5853dcdee9670e17defbcde4275fbe` "feat: introduce StatusUpdateChatInput variation"